spec/models/searchgov_domain_spec.rb

```diff
@@ -298,6 +298,17 @@
       end
     end
 
+    context 'when the domain is being indexed' do
```

Comment:
This can be replaced with the spec in Martha's branch (https://github.com/GSA/search-gov/compare/main...MothOnMars:search-gov:set_to_idle_1599) with one minor change.  It appears the defined domain in that branch doesn't error out, so I'd just recommend updating the domain to something that will error out, like:

```
      let(:searchgov_domain) do
        described_class.create!(domain: 'test.gov', activity: 'indexing')
      end
 ```

Reply:
I was trying to error out the defined domain in local machine, but the test.gov domain is resulting "200 OK" status all the time in local machine, but throwing an exception in staging. Because of this, I was unable write a proper test, need to figure that out.